CHANGELOG.md

```
@@ -1,76 +1,79 @@
+  * Dependency update: commons-lang3 -> 3.14.0
+  * Dependency update: gson -> 2.11.0
 
-#### 9.4.16 - May 2024
+#### 10.1.15 - May 2024
   * (PW-1862) Added NarrativeFragment class for detailed line information in StructuredNarrative fragments
   * Fixed SwiftMessage getPDE(): return empty value instead of null when codeword exists and has no value
   * Added isPercentage() helper method to field 37K
 
-#### 9.4.15 - March 2024
+#### 10.1.14 - March 2024
   * (PW-1812) Updated the narrative resolver, format 2 (used in field 72 for example), to allow empty values as part of the narrative fragment
```

Markdownlint

Expected: 0; Actual: 2
Unordered list indentation

(MD007, ul-indent)